### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by simonaronsson** [containrrr/watchtower#567] [General] [answered]
Welcome to the community discussion section of the watchtower repo!

@becca at GitHub has been kind enough to invite us to participate in the discussions beta, and with that - we're moving all support or discussion kind of content out from issues and into this space instead! 🎉 

Unsure whether what you're experiencing is a bug or not? Post it here first! if it is indeed a bug, moving it to issues once it's confirmed is an easy task.

**Comment by piksel** [comment, marked as answer]
So jealous! Separating discussions and questions from actual issues has been my top sought after feature for GitHub!
